src/common/Cryptography/WardenKeyGeneration.h

```diff
@@ -25,18 +25,16 @@
 class SHA1Randx
 {
 public:
-    SHA1Randx(uint8* buff, uint32 size)
+    SHA1Randx(std::array<uint8, 40> const& buf)
```

Comment:
Size of this array isnt fixed and taking half of it in code below is the correct behavior, see master branch: https://github.com/TrinityCore/TrinityCore/blob/8f9a69c9fa812b82af97809e445fc176b28f90eb/src/common/Cryptography/SessionKeyGeneration.h#L28 
It was renamed and is being used with two different sizes right now, 40 and 32